### PR TITLE
docs: fix update-tests docs

### PR DESCRIPTION
**Pull request by nicpuppa**
## Related issues

closes #

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/camunda/zeebe/compare/stable/0.24...main?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/1.3`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark

Documentation:
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] If the PR changes how BPMN processes are validated (e.g. support new BPMN element) then the Camunda modeling team should be informed to adjust the BPMN linting.

Other teams:
If the change impacts another team an issue has been created for this team, explaining what they need to do to support this change.
- [ ] [Tasklist](https://github.com/camunda/tasklist/issues)
- [ ] [Web Modeler](https://github.com/camunda/web-modeler/issues)
- [ ] [Desktop Modeler](https://github.com/camunda/camunda-modeler/issues)
- [ ] [Optimize](https://github.com/camunda/camunda-optimize/issues)

Please refer to our [review guidelines](https://github.com/camunda/zeebe/wiki/Pull-Requests-and-Code-Reviews#code-review-guidelines).

### CODE REVIEW

File: zeebe/qa/update-tests/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ## Pre-requisites
 
-To run the upgradability tests, a Docker image for Zeebe must be built with the tag 'current-test'. To do that you can run (in the zeebe-io/zeebe dir):
+To run the upgradability tests, a Docker image for Zeebe must be built with the tag 'current-test'. To do that you can run (in the camunda/zeebe dir):
 
 ```
-docker build --build-arg DISTBALL=dist/target/camunda-zeebe*.tar.gz -t camunda/zeebe:current-test --target app .
+docker build --build-arg DISTBALL='dist/target/camunda-zeebe*.tar.gz' -t camunda/zeebe:current-test --target app .
```
Comment (korthout) [inline zeebe/qa/update-tests/README.md:6]: 👍 This fixes the problem on zsh on macOS.

❓ Does it still work on linux/windows? Might be good to ask some team members to test it before merging.

**Review by korthout — APPROVED** [review]
Nice! Thanks @nicpuppa 👍 

👍 This has been an annoyance on macOS (zsh) for me for so long.

🙇 Please check with other team members whether the change works correctly on other platforms/shells.

File: zeebe/qa/update-tests/README.md (hunk 1 above)
Comment (korthout) [reply zeebe/qa/update-tests/README.md:6]: >I usually run docker commands in my Linux subsystem on Windows 🙈😄
So, no idea if this works on Windows 🤷

@tmetzke Still good to try this on your linux subsystem for windows then ;)

Comment (tmetzke) [reply zeebe/qa/update-tests/README.md:6]: Works on my Ubuntu subsystem on Windows ✔️ 

```
$ docker build --build-arg DISTBALL='dist/target/camunda-zeebe*.tar.gz' -t camunda/zeebe:current-test --target app .
[+] Building 31.6s (21/21) FINISHED                                                                                                      docker:default
 => [internal] load build definition from Dockerfile                                                                                               0.0s
 => => transferring dockerfile: 7.01kB                                                                                                             0.0s
 => resolve image config for docker.io/docker/dockerfile:1.4                                                                                       0.9s
 => CACHED docker-image://docker.io/docker/dockerfile:1.4@sha256:9ba7531bd80fb0a858632727cf7a112fbfd19b17e94c4e84ced81e24ef1a0dbc                  0.0s
 => [internal] load .dockerignore                                                                                                                  0.0s
 => => transferring context: 308B                                                                                                                  0.0s
 => [internal] load metadata for docker.io/library/ubuntu:jammy@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e            0.9s
 => [internal] load metadata for docker.io/library/eclipse-temurin:21-jdk-jammy@sha256:7a43bcc2fe4d702c572f39fe200f85d258c081ddc4da4a473ae613b033  1.2s
 => [jre-build 1/2] FROM docker.io/library/eclipse-temurin:21-jdk-jammy@sha256:7a43bcc2fe4d702c572f39fe200f85d258c081ddc4da4a473ae613b033379c9e    0.0s
 => [base 1/3] FROM docker.io/library/ubuntu:jammy@sha256:77906da86b60585ce12215807090eb327e7386c8fafb5402369e421f44eff17e                         0.0s
 => [internal] load build context                                                                                                                 24.2s
 => => transferring context: 204.14MB                                                                                                             24.1s
 => CACHED [base 2/3] RUN --mount=type=cache,target=/var/cache/apt,sharing=locked     --mount=type=cache,target=/var/lib/apt,sharing=locked     -  0.0s
 => CACHED [distball 1/3] WORKDIR /zeebe                                                                                                           0.0s
 => CACHED [jre-build 2/2] RUN jlink      --add-modules ALL-MODULE-PATH      --strip-debug      --no-man-pages      --no-header-files      --comp  0.0s
 => CACHED [java 1/3] COPY --from=jre-build /jre /opt/java/openjdk                                                                                 0.0s
 => CACHED [java 2/3] RUN java -Xshare:dump;                                                                                                       0.0s
 => CACHED [app 1/4] WORKDIR /usr/local/zeebe                                                                                                      0.0s
 => CACHED [app 2/4] RUN groupadd --gid 1001 camunda &&     adduser --system --gid 1001 --uid 1001 --home /usr/local/zeebe camunda &&     chmod g  0.0s
 => CACHED [app 3/4] COPY --link --chown=1001:0 zeebe/docker/utils/startup.sh /usr/local/bin/startup.sh                                            0.0s
 => [distball 2/3] COPY --link dist/target/camunda-zeebe*.tar.gz zeebe.tar.gz                                                                      0.5s
 => [distball 3/3] RUN mkdir camunda-zeebe &&     tar xfvz zeebe.tar.gz --strip 1 -C camunda-zeebe &&     find . -type f -name 'zbctl*' -delete    2.2s
 => [app 4/4] COPY --from=dist --chown=1001:0 /zeebe/camunda-zeebe /usr/local/zeebe                                                                0.5s
 => exporting to image                                                                                                                             1.1s
 => => exporting layers                                                                                                                            1.1s
 => => writing image sha256:0ba9f553d6d31205c0c1960573c1cb913d13cda86dc384f196f452b3e74225dd                                                       0.0s
 => => naming to docker.io/camunda/zeebe:current-test                                                                                              0.0s
```

**Pull request merged**